docs/bigcommerce-for-wordpress/getting-started/install.md

```diff
@@ -4,30 +4,30 @@
 

```

Comment:
Did you mean to edit this file as a part of this pr?

Reply:
No, that change was submitted on PR [668](https://github.com/bigcommerce/dev-docs/pull/668)